### PR TITLE
create new section for gdpr

### PR DESCRIPTION
**Pull request by emuus**
Add new section for GDPR which includes:
-get user data
-exporting data
-deleting data

### CODE REVIEW

File: admin/en/source/pages/gdpr/delete-user.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+Deleting Credentials Information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+UForge sorts certain credential information. The following may or may not exist for a given user.
+
+	* API key. Use ``apiKey_delete`` to delete an API key from the specified user API keys.
+	* Cloud Account. Use ``cloudAccount_delete`` to delete a cloud account from the specified user cloud accounts.
+	* SSH key. Use ``sshKey_delete`` to delete an ssh key from the specified user ssh keys.
+ 
+Deleting a User from a Workspace
```
Comment (jgweir) [inline admin/en/source/pages/gdpr/delete-user.rst]: shouldn't this be a title ? so requires the ~~~ marks under the text.

File: admin/en/source/pages/gdpr/overview.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.. Copyright 2017 FUJITSU LIMITED
+
+.. _gdpr-overview:
+
+GDPR Compliance
+----------------
+
+General Data Protection Regulation (GDPR) is a regulation that requires businesses to protect the personal data and privacy of EU citizens. In the context of GDPR, USS provides some basic information to help UForge administrators list user data held by UForge as well as delete the data associated to a specific user as needed.
```
Comment (jgweir) [inline admin/en/source/pages/gdpr/overview.rst]: Don't use USS, this should be UShareSoft.  We should maybe also provide the link tot he official GDPR page for further information.

File: admin/en/source/pages/gdpr/delete-user.rst (hunk 1 above)
Comment (emuus) [reply admin/en/source/pages/gdpr/delete-user.rst]: done

File: admin/en/source/pages/gdpr/overview.rst (hunk 2 above)
Comment (emuus) [reply admin/en/source/pages/gdpr/overview.rst]: done

**Pull request merged**